packages/core/src/services/documents/update.ts

```diff
@@ -69,6 +70,12 @@ export async function updateDocument({
       return Result.error(new NotFoundError('Document does not exist'))
     }
 
+    // Invalidate all resolvedContent for this commit
+    await tx
+      .update(documentVersions)
+      .set({ resolvedContent: null })
+      .where(eq(documentVersions.commitId, commitId))
+
```

Comment:
As mentioned before, all `resolvedContent` values from the same commit are invalidated when updating a document.